pkg/daemon/daemon.go

```diff
@@ -571,8 +537,9 @@ func (dn *Daemon) nodeStateSyncHandler() error {
 	// When running using systemd check if the applied configuration is the latest one
 	// or there is a new config we need to apply
 	// When using systemd configuration we write the file
-	if dn.useSystemdService {
-		systemdConfModified, err := systemd.WriteConfFile(latestState, dn.devMode, dn.platform)
+	if vars.UsingSystemdMode {
```

Comment:
shouldnt this be part of the daemon ?

tbh its not clear to me what we want to put in vars

Reply:
I put in vars everything that is used by more than one package so I don't need to start passing that around all the functions

for example this one to not pass it to the daemon and the hostInterfaces and the plugins